docs/standard/datetime/choosing-between-datetime.md

```diff
@@ -84,6 +84,23 @@ Unless a particular <xref:System.DateTime> value represents UTC, that date and t
 > [!IMPORTANT]
 > When saving or sharing <xref:System.DateTime> data, UTC should be used and the <xref:System.DateTime> value's <xref:System.DateTime.Kind%2A> property should be set to <xref:System.DateTimeKind.Utc?displayProperty=nameWithType>.
 
+## The DateOnly structure
+
+The <xref:System.DateOnly> structure represents the specific date, without time. Since it has no time component, it represents a date from the start of the day to the end of the day. This structure is ideal for storing specific dates, such as a birth date, anniversary date, or business-related dates.
```

Comment:
```suggestion
The <xref:System.DateOnly> structure represents a specific date, without time. Since it has no time component, it represents a date from the start of the day to the end of the day. This structure is ideal for storing specific dates, such as a birth date, an anniversary date, or business-related dates.
```

Reply:
I had some copy paste errors via old copies on these. I'll update them to match the other article's content.